website/README.md

```diff
@@ -0,0 +1,22 @@
+# SCIP-Java Website
+
+## Setup
+
+```bash
+$ git@github.com:sourcegraph/scip-java.git
+$ cd scip-java/
+$ yarn install
+```
+
+## Start
+
+```bash
+$ cd website/
+$ yarn run start
```

Comment:
The markdown files are generated by the sbt build
```suggestion
# In one terminal window, requires a Java installation
$ ./sbt "docs/mdoc -w"
# In a separate terminal window
$ cd website/
$ yarn run start
```

We do this for small things like having the latest version copy-pasteable from installation docs

![CleanShot 2022-07-13 at 08 24 44@2x](https://user-images.githubusercontent.com/1408093/178665101-69004cb7-f467-4666-bb8d-a0c480d151bf.png)

It's not super important so we could in theory remove it but that requires some changes that are outside the scope of this PR (which is just to document the status quo)